angular-techDraft/src/app/about/about.component.spec.ts

```diff
@@ -1,19 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { TabComponent } from './tab.component';
```

Comment:
Tab component was deleted. You are removing Tab Component.

Reply:
re added tab component, this is a change in About component